cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,66 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
+  const testData = {
+    user: {
+      username: 'Testyty',
+      password: 'M113m8J1p4z$'
+    },
+    book: {
+      name: 'Speaking JavaScript',
+      author: 'Axel Rauschmayer',
+      publisher:'O\'Reilly Media',
+      description: 'Like it or not, JavaScript is everywhere these days-from browser to server to mobile-and now you'
+    },
+  };
+
   before(() => {
-    
+    cy.visit('https://demoqa.com/login');
   });
 
-  it('', () => {
-    
+  it('login', () => {
+    cy.get('[placeholder="UserName"]')
+      .type(testData.user.username);
+    cy.get('[placeholder="Password"]')
+      .type(testData.user.password)
+    cy.contains('.btn-primary', 'Login')
+      .click();
+    cy.get('#userName-value')
+      .should('contain', testData.user.username)
+    cy.url()
+      .should('contain', '/profile');
+    });
+
+    it('Should add book', () => {
```

Comment:
Check that the cy.login() command is properly implemented in the cypress/support/commands.js file.